### PR TITLE
MUA Card RTL Migration

### PR DESCRIPTION
**Pull request by arivepr**
This PR resolves [the following JIRA card](https://issues.redhat.com/browse/RHCLOUD-24766) by:

- Adding the necessary React testing library requirements
- Adding necessary jest setup detail
- Migrating the actual component to use RTL rather than enzyme
    -Added aria label for a specific test*

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -59,7 +59,7 @@
       "<rootDir>/src/entry-dev.js",
       "<rootDir>/src/logout.js"
     ],
-    "setupFiles": [
+    "setupFilesAfterEnv": [
```
[the remark this answers is not in the document]
Comment (arivepr) [reply package.json:62]: RTL was pretty much erroring out every single test, this was the recommended change that seems to be the solution to that. 

Slows down the first pass of tests after the change, any other testing done runs just as fast as before.

File: src/test/presentional-components/myUserAccess/MUACard.test.js [hunk 2]
```diff
@@ -14,37 +12,38 @@ const entitlementsMock = [
...
-      )
-    ).toMatchSnapshot();
+  test('Should render correctly if entitled', () => {
+    render(
+      <RouterWrapper>
+        <MUACard header={mockHeader} entitlements={entitlementsMock} />
+      </RouterWrapper>
+    );
+
+    expect(screen.getByText(mockHeader)).toBeInTheDocument();
```
Comment (Hyperkid123) [inline src/test/presentional-components/myUserAccess/MUACard.test.js:25]: I think we can leave the snapshot for components like this.

**Review by Hyperkid123 — COMMENTED** [review]
Seems good, just make sure to make the snapshots again.

**Review by fhlavac — COMMENTED** [review]
Apart from resolving Martin's comments looks good to me :+1: Thank you!

**Review by Hyperkid123 — APPROVED** [review]

**Pull request merged**